api/v3/Attachment.php

```diff
@@ -493,6 +495,12 @@ function _civicrm_api3_attachment_getfields() {
     'description' => 'File content (not searchable, not returned by default)',
     'type' => CRM_Utils_Type::T_STRING,
   );
+  $spec['created_id'] = array(
+    'title' => 'Created By Contact ID',
+    'type' => CRM_Utils_Type::T_INT,
+    'description' => 'FK to civicrm_contact, who uploaded this file',
+    'api.default' => 'user_contact_id',
```

Comment:
I wonder if the File BAO is a more appropriate place to set this default value than the API? Setting it here will only affect apiV3.